README.md

```diff
@@ -95,6 +96,28 @@ Subsystem by running the following in PowerShell as an administrator
 Restart-Service LxssManager*
 ```
 
+### Local Development Login
+
+To log in to the system once locally deployed, simply enter a username in the designated field and click the "Login" button.
+
+#### Users & user types
+
+The following table lists the available usernames and their corresponding user types, as well as their associated permissions:
+
+| Username | Type of User | User Permission Scope |
```

Comment:
Since we use 'user01' in the system tests, updating this list to include the user would be helpful.

Reply:
changes made